### PR TITLE
NS-448 Restore app_boa db privileges after nightly schema refresh

### PR DESCRIPTION
**Pull request by sandeepmjay**
https://jira.ets.berkeley.edu/jira/browse/NS-448

### CODE REVIEW

File: nessie/jobs/restore_rds_user_privileges.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ENHANCEMENTS, OR MODIFICATIONS.
+"""
+
+
+from flask import current_app as app
+from nessie.externals import rds
+from nessie.jobs.background_job import BackgroundJob, BackgroundJobError
+from nessie.lib.util import resolve_sql_template
+
+"""Logic for RDS index creation job."""
```
Comment (paulkerschen) [inline nessie/jobs/restore_rds_user_privileges.py]: @sandeepmjay Copy-and-paste job description?

File: nessie/jobs/restore_redshift_user_privileges.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+"AS IS". REGENTS HAS NO OBLIGATION TO PROVIDE MAINTENANCE, SUPPORT, UPDATES,
+ENHANCEMENTS, OR MODIFICATIONS.
+"""
+
+from flask import current_app as app
+from nessie.externals import redshift
+from nessie.jobs.background_job import BackgroundJob, BackgroundJobError
+from nessie.lib.util import resolve_sql_template
+
+"""Logic for metadata schema creation job."""
```
Comment (paulkerschen) [inline nessie/jobs/restore_redshift_user_privileges.py]: Same here.

File: nessie/jobs/restore_rds_user_privileges.py (hunk 1 above)
Comment (sandeepmjay) [reply nessie/jobs/restore_rds_user_privileges.py]: yep. fixed it.

**Comment by johncrossman** [comment]
Looks good @sandeepmjay  Properly amended, it seems.

**Pull request merged**